locksmith/migrations/20220722011748-lockConstraints.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const table = 'Locks'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addConstraint(table, {
+      type: 'unique',
+      name: 'lock_address_unique_constraint',
+      fields: ['address'],
+    })
+  },
+
+  down: async (queryInterface, Sequelize) => {
+    await queryInterface.removeContraint(
+      table,
+      'lock_address_unique_constraint'
+    )
+  },
+}
```

Comment:
Maybe better to be in a separate PR as this  does seem related to upgrading the package. Same with other constraint added. 

This is adding another constraint at database level.